include/sys/arc_impl.h

```diff
@@ -446,6 +448,7 @@ typedef struct l2arc_buf_hdr {
 	uint64_t		b_daddr;	/* disk address, offset byte */
 	uint32_t		b_hits;
 	list_node_t		b_l2node;
+	arc_state_type_t	b_arcs_state;
```

Comment:
Let's instead place this 32-bit enum after `b_hits` where there's already a 32-bit hole in this structure on 64-bit systems.  This way it won't increase the size of the `l2arc_buf_hdr_t`.

Reply:
Excellent point! Thank you.